src/app/issues/PageIssues.tsx

```diff
@@ -488,48 +496,191 @@ export const PageIssues = () => {
+                                      )[100]
+                                    }
+                                    _dark={{
+                                      color: generateSwatch(
+                                        scope.color ?? brandColor
+                                      )[50],
+                                      bg: generateSwatch(
+                                        scope.color ?? brandColor
+                                      )[700],
+                                    }}
+                                  >
+                                    {scope.name}
+                                  </Tag>
+                                ))}
+
+                                {issue.scopes.length > 1 ? (
+                                  <Tooltip
+                                    hasArrow
+                                    p={2}
+                                    borderRadius={8}
+                                    bg="gray.800"
+                                    placement="bottom"
+                                    label={
+                                      <Wrap>
+                                        {issue.scopes
+                                          ?.slice(1, issue.scopes.length)
+                                          .map(({ scope }) => (
+                                            <Tag
+                                              key={scope.id}
+                                              color={
+                                                generateSwatch(
+                                                  scope.color ?? brandColor
+                                                )[700]
+                                              }
+                                              bg={
+                                                generateSwatch(
+                                                  scope.color ?? brandColor
+                                                )[100]
+                                              }
+                                              _dark={{
+                                                color: generateSwatch(
+                                                  scope.color ?? brandColor
+                                                )[50],
+                                                bg: generateSwatch(
+                                                  scope.color ?? brandColor
+                                                )[700],
+                                              }}
+                                            >
+                                              {scope.name}
+                                            </Tag>
+                                          ))}
+                                      </Wrap>
+                                    }
+                                  >
+                                    <Tag style={{ zIndex: 9999 }}>{`+${
```

Comment:
I think it's not normal to do that 😅
And for you knowledge, on Chakra UI components you a zIndex props, instead on use style -> zIndex